dojo/importers/importer/importer.py

```diff
@@ -363,6 +363,9 @@ def import_scan(self, scan, scan_type, engagement, lead, environment, active=Non
             test_import = importer_utils.update_import_history(Test_Import.IMPORT_TYPE, active, verified, tags, minimum_severity,
                                                                 endpoints_to_add, version, branch_tag, build_id, commit_hash,
                                                                 push_to_jira, close_old_findings, test, new_findings, closed_findings)
+            if apply_tags_to_findings:
+                for finding in test_import.findings_affected:
+                    finding.tags.set(tags)
```

Comment:
Using `.set` here could override tags that may have been set at the parser level. It would be better to iterate over the tags list and add them the finding that way to ensure this operation is strictly additive

Reply:
I updated the code to add the tag list instead set